File: sprint1/By languages/JS/lesson5code2.js
const time_start = Date.now();
let i = 0;
while (i < 1000000000) {
    // Ничего не делаем
    i += 1;
}
const time_finish = Date.now();
const time_span = (time_finish - time_start) / 1000; // переводим миллисекунды в секунды
console.log(time_span, 'секунд');

File: sprint1/By languages/swift/lesson5code2.swift
import Foundation

let timeStart = Date().timeIntervalSince1970
var i = 0
while i < 1_000_000_000 {
    // Do nothing
    i += 1
}

let timeFinish = Date().timeIntervalSince1970
let timeSpan = timeFinish - timeStart
print("\(timeSpan) seconds")